lib/exception_handling.rb

```diff
@@ -595,6 +596,8 @@ def send_exception_summary( exception_data, first_seen, occurrences )
         deliver(ExceptionHandling::Mailer.exception_notification(exception_data, first_seen, occurrences))
       end
     rescue StandardError, MailerTimeout => ex
+      exception_context = exception_data[:environment]
+      exception_or_string = exception_data[:error_string]
       $stderr.puts("ExceptionHandling.log_error_email rescued exception while logging #{exception_context}: #{exception_or_string}:\n#{ex.class}: #{ex}\n#{ex.backtrace.join("\n")}")
       log_error(ex, "ExceptionHandling::log_error_email rescued exception while logging #{exception_context}: #{exception_or_string}", nil, true)
```

Comment:
Feels like log_error would cause an infinite loop.  I think we would want log_info here, so that we get SOMETHING into the log file.  And don't have another issues seconds later.

FYI - I can't see an evidence of an exception having been sent to us from this line of code.  So we aren't relying on it.